_posts/2017/2017-09-01-socrates-2017-order-is-optional.md

```diff
@@ -0,0 +1,97 @@
+If those people would additionally all care about well-crafted software, you might call this gathering a [**So**ftware **Cra**ftsmanship and **Tes**ting Conference](https://www.socrates-conference.de/).
+
+# SoCraTes 2017
+
+The SoCraTes is an open space (un-)conference, meaning there is no upfront agenda: All sessions are provided by the participants based on personal experience, preference, skills, and demand.
+In the morning, sessions for the day are pitched, and the room-timeslot-matrix (see header image) is populated.
+In the evening, everyone gathers again to recap the day, and share what they have learned.
+
+This year's SoCraTes tried to be more inclusive, e.g. by providing gender-neutral bathrooms, and a field on the name tags for your preferred pronoun.
+
+I am always impressed by the openness of the attendees, everyone is sociable, and there are plenty of capable people willing to mentor.
+In this post I'd like to share what I took home from the SoCraTes 2017.
+
+## What I learned
+
+As you can guess from the conference format, the SoCraTes experience is highly individual:
+You cannot visit all sessions, and you are always free to switch sessions at any time ("Law of Two Feet").
+Prior to the conference I identified topics I would be interested in.
+Being more involved in our company's hiring process, I was really interested in how others interview candidates, on-board them, and handle project handovers.
+
+### Fixing the Pair Programming Interview
+
+Two sessions hosted by [Raimo Radczewski](https://twitter.com/rradczewski) addressed nearly all my needs.
+In the first session on Friday we discussed how to improve pair programming interviews assuming no complex domain (e.g. application or web development).
+My takeaways:
+
+* Offer the candidate a choice between a pair programming session, or a take-home exercise.
+* Before conducting the pair programming interview, write a detailed email to the candidate beforehand to introduce of yourself (the interviewer), explain the task, and list used frameworks.
+Ask whether the candidate has any special needs (e.g. dietary restrictions), and describe the development environment/machine you are going to provide to the candidate in detail.
+* A controversial point was whether we should ask a candidate to bring their own device.
+Some would like to gauge the development setup of the candidate, but it also creates an additional hurdle.
+We concluded that bringing your own device should be stated - if at all - as a secondary option.
+* During the interview, try to make it a pleasant experience, and ask them beforehand if they feel fit enough to be interviewed (similar to an exam).
+* Make sure the task is somewhat relevant to your day-to-day business or problem domain.
+Some companies use a subset of the production code base.
+Here at ePages we use the [ePizza - a microservices hands-on tutorial](https://github.com/ePages-de/epizza).
```

Comment:
I'd cut the `the` before ePizza, an alternative could be `the ePizza project - ...`